src/connectors/hits/connectHits.ts

```diff
@@ -24,16 +23,18 @@ const withUsage = createDocumentationMessageGenerator({
   connector: true,
 });
 
-export type HitsRenderState = {
+export type HitsRenderState<
+  THit extends Record<string, unknown> = Record<string, unknown>
```

Comment:
We could, but then it's exported for users too. Is that what we want?

Reply:
How do we expose public types? If we don't export them from the entry point it should be good, no?

Reply:
at the moment all types from /types are exported, so I guess we'd need to make an internal folder. However the TS recommendation is that every type you use should be exported

Reply:
We do export `BaseItem` in Autocomplete. It's useful there.

It may be less useful for InstantSearch.js final users, but for us as InstantSearch.js consumers (e.g., in VIS and RISH) it can definitely be.